src/app/backend/handler/apihandler.go

```diff
@@ -93,6 +102,42 @@ func wsMetrics(req *restful.Request, resp *restful.Response, chain *restful.Filt
 	}
 }
 
+// Post requests should set correct X-CSRF-TOKEN header, all other requests
+// should either not edit anything or be already safe to CSRF attacks (PUT
+// and DELETE)
+//
+// Validation handlers are also idempotent functions, and not actual data
+// modification opperations
+func shouldDoCsrfValidation(req *restful.Request) bool {
+	if req.Request.Method != "POST" {
+		return false
+	}
+	if strings.HasPrefix(req.SelectedRoutePath(), "/api/v1/appdeployment/validate/") {
```

Comment:
Yeah, I see, but should this be function documentation? Or implementation documentation inside the function? I'd  put it here because it stays inside the context.